src/Cache/RedisFactory.php

```diff
@@ -50,8 +50,9 @@ private function normalizeServer(string $server): array
         $parsedServer = parse_url(trim($server));
         if (! is_array($parsedServer)) {
             throw new InvalidArgumentException(sprintf(
-                'Provided server "%s" is not a valid URL with format schema://[[username]:password@]host:port',
+                'Provided server "%s" is not a valid URL with format %s',
                 $server,
+                'schema://[[username]:password@]host:port/[database]',
```

Comment:
The brackets should enclose everything which is optional, so in this case it should be `[/database]` instead of `/[database]`.

```suggestion
                'schema://[[username]:password@]host:port[/database]',
```